### PR TITLE
Hide incompatible files

### PR DESCRIPTION
**Pull request by marekkalnik**
Hi,

so this is to follow up on #29 to exclude all ignored files also from the diff to avoid confusion and reviewing unnecessary changes.

Best,

### CODE REVIEW

**Comment by leonhartX** [comment]
This seems to covered #29, so shall we close #29 and focus on this one?

**Comment by marekkalnik** [comment]
Yes, I have kept the other one in case you wanted to quick-fix the bug.

**Comment by leonhartX** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">manifest is not changed, forget to bump version?</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: src/gas-hub.js [hunk 1]
```diff
@@ -265,6 +265,11 @@ function showDiff(code, type) {
     return gasFiles.indexOf(e) < 0;
   })
   .concat(gasFiles)
+  .filter((file) => {
+    const match = file.match(/(.*?)\.(gs|html)$/);
+
```
Comment (leonhartX) [inline src/gas-hub.js]: please remove the empty line

Comment (marekkalnik) [reply src/gas-hub.js]: Sorry, I have a habit of always putting an empty line before a return ;) Fixed.

**Comment by leonhartX** [comment]
Thanks!
v2.0.5 is on the way, should be able to update soon.

**Comment by marekkalnik** [comment]
Great, thanks ! :)

**Pull request merged**